plugins/logging/pkg/gateway/stream.go

```diff
@@ -0,0 +1,17 @@
+package gateway
+
+import (
+	"github.com/rancher/opni/pkg/capabilities/wellknown"
+	streamext "github.com/rancher/opni/pkg/plugins/apis/apiextensions/stream"
+	"github.com/rancher/opni/plugins/logging/pkg/apis/node"
+)
+
+func (p *Plugin) StreamServers() []streamext.Server {
+	return []streamext.Server{
+		{
+			Desc:              &node.NodeLoggingCapability_ServiceDesc,
+			Impl:              &p.logging,
+			RequireCapability: wellknown.CapabilityMetrics,
```

Comment:
Metrics -> Logging (although this field is currently unused, but it will be used later)